README.md

```diff
@@ -1,6 +1,4 @@
-# @celestiaorg/CIPs
-
-## Celestia Improvement Proposal (CIP) process repository
+# Celestia Improvement Proposal (CIP) process repository
```

Comment:
[optional] most other Github repos don't have "repository" in the top-level heading. "process" also seems unnecessary here.

```suggestion
# Celestia Improvement Proposal (CIP)
```